modules/template/src/main/twirl/scaladex/view/project/versions.scala.html

```diff
@@ -0,0 +1,143 @@
+          <div>We can't find any release that matches all you criteria</div>
+        } else {
+        <div>
+          @artifactByVersions.map { case ((releaseDate, version), artifacts) =>
+          <div class="panel">
+            <div class="artifact-row panel-body center">
+              <div class="col-md-3 center">
+                <a class="artifact-version" href="/@project.reference/artifacts/@artifactName/@version">@version</a>
+              </div>
+              <div class="col-md-9">
+                <div class="artifact-info">
+                  <a data-toggle="tooltip" data-placement="top" title="@formatInstant(releaseDate)">
+                    @TimeUtils.toFiniteDuration(releaseDate, Instant.now()).prettyPrint
+                    ago<i class="fa-solid fa-calendar-days"></i>
+                  </a>
+                  <br>
+                  @artifacts.head.licenses.map { license =>
+                    <a href="@license.url.getOrElse("#")">@license.shortName<i class="fa-solid fa-scale-balanced"></i></a>
+                  }
+                </div>
+                <div class="artifact-content">
+                  @for(
+                    (platform, binaryVersions) <- artifacts
+                      .map(_.binaryVersion)
+                      .distinct
+                      .sorted
+                      .reverse
+                      .groupBy(_.platform)
+                      .toSeq
+                  ) {
+                    <h5>@platform:</h5>
+                    @for(binaryVersion <- binaryVersions) { 
+                      <a href="/@project.reference/artifacts/@artifactName/@version?binary-version=@binaryVersion.encode" class="scala-version">
+                        @binaryVersion.language.label
+                      </a>
+                    }
+                    <br>
+                  }
+                </div>
+              </div>
+            </div>
+          </div>
+          }
+        }
+        </div>
+      </div>
+    </div>
+  </main>
+}
+
+@filters() = {
+  <form action="#" action="GET">
+    <select class="selectpicker" name="binary-versions" 
+      title= "Scala Versions" multiple data-style="btn-primary" data-actions-box="true" data-selected-text-format="static"
+      onchange="this.form.submit()">
+      @for((platform, binaryVersions) <- allBinaryVersions.sorted.groupBy(_.platform)) {
+        <optgroup label="@platform">
+          @for(binaryVersion <- binaryVersions.reverse) {
+            <option value="@binaryVersion.label"
+            @if(params.binaryVersions.contains(binaryVersion)) {selected}>
+            @binaryVersion.language.label
+            </option>
+          }
+        </optgroup>
+      }
+    </select>
+    <label class="filter btn btn-default">
+      <input type="checkbox"  @if(params.preReleases) { checked } name="pre-releases" value="true" onclick="this.form.submit()">
+      Show pre-release versions
+    </label>
+  </form>
+}
```

Comment:
Those filters are the same in 2 pages so we can extract them as a `artifacts-filters.scala.html` file. Same for the badges?